docs/narr/startup.rst

```diff
@@ -132,7 +132,7 @@ Here's a high-level time-ordered overview of what happens when you press
 #. ``pserve`` starts the WSGI *server* defined within the ``[server:main]``
    section.  In our case, this is the Waitress server (``use =
    egg:waitress#main``), and it will listen on all interfaces on port 6543
-   for both IPv4 and IPv6 (``listen = 127.0.0.1:6543 [::1]:6543``). The server
+   for both IPv4 and IPv6 (``listen = localhost:6543``). The server
```

Comment:
FYI: due to some router woes, I always had to remove the IPv6 argument in my local setups. That environment is IPv4 only, and attempting to bind IPv6 explicitly resulted in plain start-up failures. Therefore this change will remove setup hassle for certain environments.